axelor-bank-payment/src/main/java/com/axelor/apps/bankpayment/service/bankorder/file/directdebit/BankOrderFile00800101Service.java

```diff
@@ -434,7 +434,8 @@ protected void createDrctDbtTxInf(
         company = bankOrderLine.getBankOrder().getSenderCompany();
       }
       Umr receiverUmr =
-          Beans.get(UmrService.class).getActiveUmr(company, bankOrderLine.getPartner());
+          Beans.get(UmrService.class)
+              .getActiveUmr(bankOrderLine.getReceiverCompany(), receiverBankDetails);
```

Comment:
We should improve the exception message a few lines below : BankPaymentExceptionMessage.DIRECT_DEBIT_MISSING_PARTNER_ACTIVE_UMR

Currently this message is : "Please add an Active UMR to the partner."
Change it for : "Please add an Active UMR to the partner %s for the bank details %s."